aixplain/modules/benchmark_job.py

```diff
@@ -92,3 +92,70 @@ def download_results_as_csv(self, save_path: Optional[Text] = None, return_dataf
             error_message = f"Downloading Benchmark Results: Error in Downloading Benchmark Results : {e}"
             logging.error(error_message, exc_info=True)
             raise Exception(error_message)
+        
+    def __simplify_scores(self, scores):
+        simplified_score_list  = []
+        for model_id, model_info in scores.items():
+            model_scores = model_info["rawScores"]
+            # model = Mode
+            row = {"Model": model_id}
+            for score_info in model_scores:
+                row[score_info["longName"]] = score_info["average"]
```

Comment:
Make sure these fields are always there. Does it make sense to add a try/catch to treat when they are not?

Reply:
These are mandatory fields and will always be there. In case of any issue, they are sent as None instead but the key will always be there